lib/__tests__/InstantSearch-test.js

```diff
@@ -176,4 +176,54 @@ describe('InstantSearch lifecycle', () => {
       expect(algoliasearchHelper.args[0][2].values).toEqual([-2, -1, 0, 1, 3, 4, 2]);
     });
   });
+
+  context('when render happens', () => {
+    var render = sinon.spy();
+    beforeEach(() => {
+      render.reset();
```

Comment:
yes you can use reset but jyou could also assign render only in beforeEach which result in the same (but avoid initiliazing it and then reset it). That's the same effect so don't care